lib/scenarios.rb

```diff
@@ -13,6 +13,7 @@ class Scenarios < BaseUtil
   PLANS_UQHP_FAMILY = 'plans_for_uqhp_family'
   PLANS_CSR_FAMILY = 'plans_for_family_receiving_csr'
   RIDP = 'ridp'
+  USER_EXISTENCE = 'user_existence'
 
```

Comment:
Based on where we are heading with this PR #10, I suspect we are going to redo a lot of this anyways so perhaps we can dump this PR as well, and deal with it separately once we have a handle on how we want to do this (_hopefully, after we meet to discuss this_)?